index.html

```diff
@@ -590,6 +590,9 @@ <h1>How Can I Participate in FOSSASIA projects?</h1>
                 				<li><a target="_self" href="https://www.instagram.com/explore/tags/fossasia/"><i class="icon social_instagram"></i></a></li>
 								<li><a target="_self" href="https://github.com/fossasia"><i class="fa fa-github fa-lg"></i></a></li>
 								<li><a target="_self" href="https://gitlab.com/fossasia"><i class="fa fa-gitlab fa-lg"></i></a></li>
+								<li><a target="_self" href="https://www.weibo.com/u/5784920339"><i class="fa fa-weibo fa-lg"></i></a></li>
+								<li><a target="_self" href="https://vk.com/fossasia"><i class="fa fa-vk fa-lg"></i></a></li>
+								<li><a target="_self" href="https://de.m.wikipedia.org/wiki/FOSSASIA"><i class="fa fa-wikipedia-w fa-lg"></i></a></li>
```

Comment:
remove this. we need an english article

Reply:
@cweitat what's wrong?

Reply:
this is a german wiki. best to link english wiki instead